### DISCUSSION TITLE
Add character limits to Text widgets and Title widgets in the advance settings

### DISCUSSION THREAD

**Discussion by portrayaloflife** [elementor/elementor#29523] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Build a nice layout and the client with editor access pastes a whole paragraph where a sentence should go!

### Describe the solution you'd like.

Add an option for character limits so we can help add guardrails to design templates. 

### Describe alternatives you've considered.

Plugins

### Additional context

Would be awesome! 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by hitsgee** [comment]
I get around this using ACF but a native restriction would be great.
